tests/acceptance/common.py

```diff
@@ -52,11 +52,7 @@ def _start_qemu(qenv=None, conn=None):
     if qenv:
         env.update(qenv)
 
-    proc = subprocess.Popen(
-        ["../../meta-mender-qemu/scripts/mender-qemu"],
-        env=env,
-        start_new_session=True,
-    )
+    proc = subprocess.Popen([qemu_wrapper], env=env, start_new_session=True,)
```

Comment:
wait, so what if `qemu_wrapper` is None here?

Reply:
Good question. I will modify the method signature to make all arguments mandatory.